bin/cli.js

```diff
@@ -32,6 +20,37 @@ const logLevels = {
   debug: 4,
 };
 
+const args = yargs
+  .usage('$0 [options] file.js [file.js...]')
+  .option('exclude', {
+    array: true,
+    type: 'string',
+    describe: 'Exclude tests patterns',
+  })
+  .option('reporter', {
+    type: 'string',
+    describe: 'Reporter name',
+  })
+  .option('log-level', {
+    choices: Object.keys(logLevels),
+    type: 'string',
+    describe: 'Log level',
+  })
+  .option('run-todo', {
+    type: 'boolean',
+    describe: 'Run todo tests',
+  })
+  .option('exit-timeout', {
+    type: 'number',
+    describe: 'Seconds to wait after tests finished',
+  })
+  .option('config', {
+    alias: 'c',
+    coerce: conf => (Array.isArray(conf) ? conf[conf.length - 1] : conf),
```

Comment:
There is an option to disable this behavior - https://github.com/yargs/yargs-parser#duplicate-arguments-array.
I think we can disable it globally.